### ISSUE TITLE
In AWS private agents are set with public IP address

### ISSUE THREAD

**Issue by beeyeas** [bernadinm/terraform-dcos#12]
I was successful in using terraform apply to create my DC/OS Instance .
But i notice private agents are configured with public ip address.

**Comment by bernadinm** [comment]
Thats correct, though they are configured with a public address there are security groups in place which locks down the access from the outside world. The only have egress access to the internet but no ingress other than ssh. Let me know if you have any other questions.

**Comment by hantuzun** [comment]
It's the case for Azure as well. I guess this's done in order to be able to run provisioners through their assigned IP addresses?

**Comment by bernadinm** [comment]
@hantuzun, that is exactly correct. This is not the only way to achieve this. Another method is to make use of bastion host to have the provisioners go through but each one has their pros and cons.
